client/src/App.tsx

```diff
@@ -22,6 +23,7 @@ function App() {
     <div>
       <Routes>
         <Route path="/" element={<NavBar />}>
+          <Route index />
```

Comment:
What's the purpose of this route? It doesn't have an element, so it seems it wouldn't do anything.